phaser/cli/main.py

```diff
@@ -68,6 +69,11 @@ def main(argv):
         help="output more information during execution",
         action="store_true",
     )
+    parser.add_argument(
+        "-l", "--log",
+        help="set the log level",
+        default="INFO"
```

Comment:
It would be helpful to to use `choices` to limit what log levels can be set and to have better documentation in the CLI usage output.  https://docs.python.org/3/library/argparse.html#choices